subsys/bluetooth/controller/Kconfig

```
@@ -309,6 +312,21 @@ config BT_CTLR_TX_PWR_MINUS_40
 
 endchoice
 
+config BT_CTLR_TX_PWR_SET
+	int "Set TX power (dBm)"
+	range -127 127
+	default 0
+	depends on BT_CTLR_TX_PWR_0
```

Why does it depend on `BT_CTLR_TX_PWR_0`?

This feature is not supported by Zephyr's LL.
```suggestion
	depends on BT_CTLR_TX_PWR_0 && !BT_LL_SW_SPLIT
```

Shouldn't it depend on the `BT_CTLR_TX_PWR_DYNAMIC_CONTROL`?
There is no actual use case in this PR (also no reference for any), hence I just try to figure out any possible. 

It looks like the feature sets default TX power. It seems that it is achieved by similar functionality that Zephyrs LL uses for dynamic power control. The difference is that Zephyr's LL does not include FEM into its algorithm.

---

It depends on `BT_CTLR_TX_PWR_0` so that `BT_CTLR_TX_PWR` choices take precedence over this kconfig.
`BT_CTLR_TX_PWR_DYNAMIC_CONTROL` is not necessary for this and they can work together.
I will add `!BT_LL_SW_SPLIT` as a dependency.